src/main/java/de/fearnixx/jeak/service/controller/testImpls/TestController.java

```diff
@@ -6,7 +6,7 @@
 import de.fearnixx.jeak.service.controller.reflect.RequestParam;
 import de.fearnixx.jeak.service.controller.reflect.RestController;
 
-@RestController(pluginId = "pid", endpoint = "/test")
+@RestController(pluginId = "default", endpoint = "/test")
```

Comment:
"default" :sweat_smile:

Reply:
Just some random name for the pluginId

Reply:
If that doesn't decide anything, that's fine. But I would favor changing it to something more meaningful.

Reply:
Done